### PR TITLE
Dev branch

### PR DESCRIPTION
**Pull request by scshitole**
Node resources changes, introduced field interval and updated the docs.

### CODE REVIEW

**Comment by dannyk81** [comment]
@scshitole have you had a chance to look at the comments?

File: bigip/resource_bigip_ltm_node.go [hunk 1]
```diff
@@ -211,7 +210,6 @@ func resourceBigipLtmNodeUpdate(d *schema.ResourceData, meta interface{}) error
```
Comment (dannyk81) [inline bigip/resource_bigip_ltm_node.go]: Please remove `d.SetId("")` from line 202, Exists function should not modify state.

https://github.com/terraform-providers/terraform-provider-bigip/issues/7

File: bigip/resource_bigip_ltm_node.go [hunk 2]
```diff
@@ -183,6 +180,8 @@ func resourceBigipLtmNodeRead(d *schema.ResourceData, meta interface{}) error {
 
 	d.Set("connection_limit", node.ConnectionLimit)
 	d.Set("dynamic_ratio", node.DynamicRatio)
+	d.Set(("fqdn.0.interval"), node.FQDN.Interval)
```
Comment (dannyk81) [inline bigip/resource_bigip_ltm_node.go]: This is an element in an aggregate type, I believe this should be error checked when refreshing state.

File: bigip/resource_bigip_ltm_node.go [hunk 3]
```diff
@@ -235,8 +233,7 @@ func resourceBigipLtmNodeUpdate(d *schema.ResourceData, meta interface{}) error
```
Comment (dannyk81) [inline bigip/resource_bigip_ltm_node.go]: You are missing the interval attribute in the Update function when it's an FQDN Node.

**Comment by scshitole** [comment]
will create a new pull request

**Pull request closed without merging**